### PR TITLE
Organize Chinese docs

### PR DESCRIPTION
**Pull request by LuQQiu**
This PR does not add any new content to Chinese docs, but re-organize the docs.

Move the content of advanced/Time-to-Live.md into Caching.md
Move the content of contributor/Logging-Conventions.md into operation/Server-Logging.md
Move the content of storage/secure-HDFS.md into storage/HDFS.md

Rename ufs folder to storage

Move docs/cn/FAQ.md → docs/cn/overview/FAQ.md
Move docs/cn/Getting-Started.md → docs/cn/overview/Getting-Started.md

### CODE REVIEW

File: docs/cn/ufs/Ufs-Extension-API.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
 ---
 layout: global
-title: 如何开发底层存储扩展
-group: Client APIs
-priority: 3
+title: 新底层存储系统扩展软件
+nickname: 创建新底层存储系统
+group: Storage Integrations
+priority: 101
```
Comment (jiacheliu3) [inline docs/cn/ufs/Ufs-Extension-API.md]: what is 101?

File: docs/cn/ufs/Ufs-Extension-API.md [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 ---
 
 * 内容列表
 {:toc}
 
-该页是面向底层存储扩展的开发者的。请浏览[managing extensions](UFSExtensions.html)来获取使用已有扩展的指导。
+该页是面向底层存储扩展的开发者的。请浏览[管理使用新底层存储系统扩展软件文档]({{ '/cn/storage/Ufs-Extension.html' | relativize_url }})来获取使用已有扩展的指导。
```
Comment (jiacheliu3) [inline docs/cn/ufs/Ufs-Extension-API.md]: "管理使用新底层存储系统扩展软件文档" -> "管理使用新底层存储系统扩展文档"?
Actually i can't read the whole phrase in one go anyway

Comment (LuQQiu) [reply docs/cn/ufs/Ufs-Extension-API.md]: Done
it's hard to translate ufs extension ...

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**